Management/src/ActuatorWeb/http-client.env.json

```diff
@@ -0,0 +1,15 @@
+{
+  "localhttp-dedicatedactuatorport-basicauth": {
+    "AppsManagerOrigin": "https://your-apps-manager.origin",
+    "HostAddress": "http://localhost:5126",
+    // See the note about HTTPS in the ActuatorWeb README
```

Comment:
I don't see the point of adding this comment here. The environment is http-only, so using non-http here would require an explanation.